draft-ietf-rtcweb-jsep.xml

```diff
@@ -697,8 +697,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           control how aggressively it will try to BUNDLE media streams together.
           The set of available policies is as follows:
           <list style="hanging">
-              <t hangText="balanced:">The application will BUNDLE all media streams of
-              the same type together. That is, if there are multiple audio and
+              <t hangText="balanced:">The application will bundle all media streams into
+              a single bundle group. All but the first media stream of each
+              type will be marked as bundle-only. That is, if there are multiple audio and
```

Comment:
---
The offer includes a single bundle group across all media sections.  The first media section of each type (audio, video, or application) MUST(?) contain transport parameters, which will allow an answerer to unbundle that section.  The second and any subsequent media section of each type will be marked bundle-only. 

Consequently, candidates are gathered for at most three media sections.  If there are multiple streams of a video type, the only option given to an answerer is to accept the bundling of those streams, or reject them entirely.

When negotiating with a non-BUNDLE aware [...]
---
I note that if there is an offer with two audio and two video streams, unbundling the first video will leave the answerer with a choice of bundling the second video with the audio, or rejecting it.  That's shitty, but I guess that's the design we have to deal with